desk/sur/chat.hoon

```diff
@@ -1,7 +1,7 @@
 /-  g=groups, zer=chat-0
```

Comment:
> So we definitely need to remove the %load case from chat, and then are we nuking binnec again instead of migrating those changes? If we are then this is good to go after fixing chat types. Also was %load the reason the watch-acks failed? I don't know if we ever explicitly stated anywhere.

 the watch acks weren't harmful actually, was just that swapped line. i was able to trigger the watch ack failure for the case that i was worried about and i'm confident they aren't harmful. would be nice to get rid of them though.

%load is removed now, definitely need a thorough review on the chat changes  though.

Reply:
I'm not seeing load removed. where did you mean?

Reply:
https://github.com/tloncorp/landscape-apps/pull/1031/files/f761a033e195e2d341528659c046c2052c41ebbb#diff-9c76c0009e3d5ee93a5aefccb8c8c6777723e4c6c99dabb5b24891fae8466d0aR232

Reply:
I meant in chat but I saw it in your other PR so I think they just need to get merged together